2013-02-02T17:22:46+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(74): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(209): App_Router->route()
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-02T17:24:59+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(74): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(209): App_Router->route()
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-02T17:26:21+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(75): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(209): App_Router->route()
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-02T17:26:44+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(75): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(209): App_Router->route()
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-02T17:27:48+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(75): App_Router_Router->findRoute('/default-app/in...')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(209): App_Router->route()
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-02T17:45:12+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(75): App_Router_Router->findRoute('/default-app/in...')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(209): App_Router->route()
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(28): App::run()
#3 {main}
2013-02-02T17:45:18+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(75): App_Router_Router->findRoute('/default-app/in...')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(209): App_Router->route()
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(28): App::run()
#3 {main}
2013-02-02T22:21:33+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute(NULL)
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(267): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(27): App::run()
#3 {main}
2013-02-07T17:01:41+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/mo...')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(270): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(27): App::run()
#3 {main}
2013-02-09T18:02:21+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(270): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(27): App::run()
#3 {main}
2013-02-09T19:26:29+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(27): App::run()
#3 {main}
2013-02-09T20:23:03+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-09T20:29:48+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-09T20:37:27+00:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: test' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/View.php:42
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Controller.php(87): App_View->loadView('moep', 'test')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php(110): App_Controller->__loadView()
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php(90): App_Router_Dispatcher->dispatchController('moep', 'test', Array, NULL)
#3 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#5 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#6 {main}
2013-02-09T20:37:27+00:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: test' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/View.php:45
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Controller.php(87): App_View->loadView('moep', 'test')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php(110): App_Controller->__loadView()
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php(90): App_Router_Dispatcher->dispatchController('moep', 'test', Array, NULL)
#3 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#5 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#6 {main}
2013-02-09T20:39:05+00:00 ERR (3): exception 'App_Exception' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/View.php:42
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Controller.php(87): App_View->loadView('moep', 'test')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php(110): App_Controller->__loadView()
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php(90): App_Router_Dispatcher->dispatchController('moep', 'test', Array, NULL)
#3 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#5 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#6 {main}
2013-02-09T20:39:05+00:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: test' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/View.php:45
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Controller.php(87): App_View->loadView('moep', 'test')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php(110): App_Controller->__loadView()
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php(90): App_Router_Dispatcher->dispatchController('moep', 'test', Array, NULL)
#3 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#5 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#6 {main}
2013-02-09T20:48:39+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/mo...')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-09T20:48:45+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-09T20:48:48+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-09T20:48:54+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-09T20:57:49+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-09T21:00:29+00:00 ERR (3): exception 'methodNotSpecifiedException' with message 'Method Name not specified' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php:56
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-09T21:01:31+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-09T21:03:18+00:00 ERR (3): exception 'classMethodNotFoundException' with message 'method not found ' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php:88
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-09T21:03:28+00:00 ERR (3): exception 'classMethodNotFoundException' with message 'method not found ' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php:88
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-09T21:04:19+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-09T21:04:24+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-09T21:04:31+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-09T21:04:34+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(259): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-19T21:44:11+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(255): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-19T21:51:25+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(255): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-19T21:55:34+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(255): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-19T21:55:54+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(255): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-19T21:56:09+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(255): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-19T21:56:54+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(255): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-19T21:57:03+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(255): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-19T21:57:17+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(255): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-19T21:57:17+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(255): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-19T21:59:33+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(255): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(26): App::run()
#3 {main}
2013-02-19T22:01:39+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(255): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(25): App::run()
#3 {main}
2013-02-19T22:02:43+00:00 ERR (3): exception 'RouteNotFoundException' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Router.php:138
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(255): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(24): App::run()
#3 {main}
2013-02-22T11:36:46+01:00 ERR (3): exception 'RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:138
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(255): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T11:46:20+01:00 ERR (3): exception 'RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:139
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(255): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T11:59:45+01:00 ERR (3): exception 'RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:139
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:04:02+01:00 ERR (3): exception 'RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:138
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Router->findRoute(Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:24:11+01:00 ERR (3): exception 'RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:141
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Router->findRoute(Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:27:37+01:00 ERR (3): exception 'RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:141
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Router->findRoute(Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:27:45+01:00 ERR (3): exception 'RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:141
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Router->findRoute(Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:28:32+01:00 ERR (3): exception 'RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:141
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Router->findRoute(Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:28:53+01:00 ERR (3): exception 'RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:141
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Router->findRoute(Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:29:19+01:00 ERR (3): exception 'RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:141
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Router->findRoute(Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:29:22+01:00 ERR (3): exception 'RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:141
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Router->findRoute(Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:29:32+01:00 ERR (3): exception 'RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:141
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Router->findRoute(Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:33:15+01:00 ERR (3): exception 'classMethodNotFoundException' with message 'method not found index' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:88
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(112): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:34:01+01:00 ERR (3): exception 'classMethodNotFoundException' with message 'method not found index' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:88
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(112): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:34:02+01:00 ERR (3): exception 'classMethodNotFoundException' with message 'method not found index' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:88
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(112): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:34:56+01:00 ERR (3): exception 'classMethodNotFoundException' with message 'method not found index' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:88
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(112): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:35:00+01:00 ERR (3): exception 'RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:141
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(111): App_Router_Router->findRoute(Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:35:07+01:00 ERR (3): exception 'classMethodNotFoundException' with message 'method not found index' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:88
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(112): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:37:05+01:00 ERR (3): exception 'classMethodNotFoundException' with message 'method not found index' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:88
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:38:13+01:00 ERR (3): exception 'App_Exception' in C:\xampp\htdocs\default-app\lib\App\View.php:42
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(87): App_View->loadView(NULL, NULL)
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(110): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(92): App_Router_Dispatcher->dispatchController('index', 'index', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T12:38:13+01:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: ' in C:\xampp\htdocs\default-app\lib\App\View.php:45
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(87): App_View->loadView(NULL, NULL)
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(110): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(92): App_Router_Dispatcher->dispatchController('index', 'index', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T12:38:19+01:00 ERR (3): exception 'classMethodNotFoundException' with message 'method not found index' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:88
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:38:55+01:00 ERR (3): exception 'classMethodNotFoundException' with message 'method not found index' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:93
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:39:08+01:00 ERR (3): exception 'RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:141
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(109): App_Router_Router->findRoute(Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:39:26+01:00 ERR (3): exception 'classMethodNotFoundException' with message 'method not found index' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:93
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:39:31+01:00 ERR (3): exception 'classMethodNotFoundException' with message 'method not found index' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:93
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:42:17+01:00 ERR (3): exception 'classMethodNotFoundException' with message 'method not found index' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:93
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T12:43:46+01:00 ERR (3): exception 'App_Exception' in C:\xampp\htdocs\default-app\lib\App\View.php:42
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(87): App_View->loadView(NULL, NULL)
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(117): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(99): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T12:43:46+01:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: ' in C:\xampp\htdocs\default-app\lib\App\View.php:45
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(87): App_View->loadView(NULL, NULL)
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(117): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(99): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T12:44:24+01:00 ERR (3): exception 'App_Exception' in C:\xampp\htdocs\default-app\lib\App\View.php:42
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(87): App_View->loadView(NULL, NULL)
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(115): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(97): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T12:44:24+01:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: ' in C:\xampp\htdocs\default-app\lib\App\View.php:45
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(87): App_View->loadView(NULL, NULL)
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(115): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(97): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:08:25+01:00 ERR (3): exception 'App_Exception' in C:\xampp\htdocs\default-app\lib\App\View.php:42
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(87): App_View->loadView(NULL, NULL)
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(115): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(97): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:08:25+01:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: ' in C:\xampp\htdocs\default-app\lib\App\View.php:45
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(87): App_View->loadView(NULL, NULL)
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(115): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(97): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:08:46+01:00 ERR (3): exception 'App_Exception' in C:\xampp\htdocs\default-app\lib\App\View.php:42
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(87): App_View->loadView(NULL, NULL)
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(115): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(97): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:08:46+01:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: ' in C:\xampp\htdocs\default-app\lib\App\View.php:45
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(87): App_View->loadView(NULL, NULL)
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(115): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(97): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:09:32+01:00 ERR (3): exception 'App_Exception' in C:\xampp\htdocs\default-app\lib\App\View.php:42
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(87): App_View->loadView(NULL, NULL)
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(115): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(97): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:09:32+01:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: ' in C:\xampp\htdocs\default-app\lib\App\View.php:45
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(87): App_View->loadView(NULL, NULL)
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(115): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(97): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(110): App_Router_Dispatcher->dispatch(Object(App_Router_Route))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:17:14+01:00 ERR (3): exception 'App_Exception' in C:\xampp\htdocs\default-app\lib\App\View.php:42
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(87): App_View->loadView(NULL, NULL)
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(118): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(100): App_Router_Dispatcher->dispatchController('index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:17:14+01:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: ' in C:\xampp\htdocs\default-app\lib\App\View.php:45
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(87): App_View->loadView(NULL, NULL)
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(118): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(100): App_Router_Dispatcher->dispatchController('index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:45:01+01:00 ERR (3): exception 'App_Exception' in C:\xampp\htdocs\default-app\lib\App\View.php:42
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(93): App_View->loadView('index', 'index')
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(117): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(99): App_Router_Dispatcher->dispatchController('index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:45:01+01:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: index' in C:\xampp\htdocs\default-app\lib\App\View.php:45
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(93): App_View->loadView('index', 'index')
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(117): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(99): App_Router_Dispatcher->dispatchController('index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:45:17+01:00 ERR (3): exception 'App_Exception' in C:\xampp\htdocs\default-app\lib\App\View.php:42
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(93): App_View->loadView('Index', 'index')
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(117): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(99): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:45:17+01:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: index' in C:\xampp\htdocs\default-app\lib\App\View.php:45
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(93): App_View->loadView('Index', 'index')
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(117): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(99): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:45:23+01:00 ERR (3): exception 'App_Exception' in C:\xampp\htdocs\default-app\lib\App\View.php:42
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(93): App_View->loadView('index', 'index')
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(117): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(99): App_Router_Dispatcher->dispatchController('index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:45:23+01:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: index' in C:\xampp\htdocs\default-app\lib\App\View.php:45
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(93): App_View->loadView('index', 'index')
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(117): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(99): App_Router_Dispatcher->dispatchController('index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:45:57+01:00 ERR (3): exception 'App_Exception' in C:\xampp\htdocs\default-app\lib\App\View.php:42
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(93): App_View->loadView('index', 'index')
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(117): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(99): App_Router_Dispatcher->dispatchController('index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:45:57+01:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: index' in C:\xampp\htdocs\default-app\lib\App\View.php:45
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(93): App_View->loadView('index', 'index')
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(117): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(99): App_Router_Dispatcher->dispatchController('index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:48:13+01:00 ERR (3): exception 'App_Exception' in C:\xampp\htdocs\default-app\lib\App\View.php:43
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(93): App_View->loadView('index', 'index')
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(117): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(99): App_Router_Dispatcher->dispatchController('index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T14:48:13+01:00 ERR (3): exception 'App_Exception' with message 'Could not load view from action: index' in C:\xampp\htdocs\default-app\lib\App\View.php:46
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Controller.php(93): App_View->loadView('index', 'index')
#1 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(117): App_Controller->__loadView()
#2 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(99): App_Router_Dispatcher->dispatchController('index', 'indexAct', Array, NULL)
#3 C:\xampp\htdocs\default-app\lib\App\Router.php(111): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#4 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#5 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#6 {main}
2013-02-22T16:25:29+01:00 ERR (3): exception 'Router_ClassNoInstanceOfController' with message 'IndexIs not an instance of APP_Controller' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:114
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(92): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array)
#1 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#2 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#3 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#4 {main}
2013-02-22T16:25:46+01:00 ERR (3): exception 'Router_ClassNoInstanceOfController' with message 'Index is not an instance of APP_Controller' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:114
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(92): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array)
#1 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#2 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#3 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#4 {main}
2013-02-22T16:25:59+01:00 ERR (3): exception 'Router_ClassNoInstanceOfController' with message 'Class Index is not an instance of APP_Controller' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:114
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(92): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array)
#1 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#2 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#3 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#4 {main}
2013-02-22T16:26:20+01:00 ERR (3): exception 'Router_ClassNoInstanceOfController' with message 'Class Index is not an instance of APP_Controller' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:114
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(92): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array)
#1 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#2 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#3 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#4 {main}
2013-02-22T16:27:06+01:00 ERR (3): exception 'Router_ClassNoInstanceOfController' with message 'Class Index is not an instance of APP_Controller' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:114
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(92): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array)
#1 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#2 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#3 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#4 {main}
2013-02-22T16:27:19+01:00 ERR (3): exception 'Router_ClassNoInstanceOfController' with message 'Controller Index is not an instance of APP_Controller' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:114
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(92): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array)
#1 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#2 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#3 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#4 {main}
2013-02-22T16:27:28+01:00 ERR (3): exception 'Router_ClassNoInstanceOfController' with message 'Index is not an instance of APP_Controller' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:114
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(92): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array)
#1 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#2 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#3 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#4 {main}
2013-02-22T16:33:35+01:00 ERR (3): exception 'Router_NoAppControllerException' with message 'Index is not an instance of APP_Controller' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:114
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(92): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array)
#1 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#2 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#3 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#4 {main}
2013-02-22T16:34:41+01:00 ERR (3): exception 'Router_NoAppControllerException' with message 'Index is not an instance of APP_Controller' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:114
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(92): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array)
#1 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#2 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#3 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#4 {main}
2013-02-22T16:42:24+01:00 ERR (3): exception 'Router_ClassFileNotFoundException' with message 'Class file not found' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:78
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T16:42:46+01:00 ERR (3): exception 'Router_ClassFileNotFoundException' with message 'Class file not found' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:78
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T16:42:54+01:00 ERR (3): exception 'Router_ClassMethodNotFoundException' with message 'method not found sgsdgsdgAct' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:88
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T16:43:04+01:00 ERR (3): exception 'Router_ClassMethodNotFoundException' with message 'method not found sgsdgsdgAct' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:88
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T16:46:14+01:00 ERR (3): exception 'Router_ClassMethodNotFoundException' with message 'method not found asasfAct' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:86
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T16:46:18+01:00 ERR (3): exception 'Router_ClassMethodNotFoundException' with message 'method not found wegAct' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:86
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T16:46:24+01:00 ERR (3): exception 'Router_ClassMethodNotFoundException' with message 'method not found rgergAct' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:86
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(116): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T17:38:24+01:00 ERR (3): exception 'Router_ClassMethodNotFoundException' with message 'method not found dfgdAct' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:86
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(117): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T17:41:34+01:00 ERR (3): exception 'Router_ClassMethodNotFoundException' with message 'method not found dfgdAct' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:86
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(117): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-02-22T17:43:12+01:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:136
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(115): App_Router_Router->findRoute('/default-app\mo...')
#1 C:\xampp\htdocs\default-app\lib\App\View.php(67): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\app\views\index\index.phtml(1): App_View->action('test', 'moep')
#3 C:\xampp\htdocs\default-app\lib\App\View.php(40): require_once('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\default-app\lib\App\Controller.php(94): App_View->loadView('index', 'index')
#5 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(109): App_Controller->__loadView()
#6 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(90): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array)
#7 C:\xampp\htdocs\default-app\lib\App\Router.php(117): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#8 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#9 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#10 {main}
2013-02-22T17:44:09+01:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:136
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(115): App_Router_Router->findRoute('/default-app\mo...')
#1 C:\xampp\htdocs\default-app\lib\App\View.php(69): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\app\views\index\index.phtml(1): App_View->action('test', 'moep')
#3 C:\xampp\htdocs\default-app\lib\App\View.php(40): require_once('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\default-app\lib\App\Controller.php(94): App_View->loadView('index', 'index')
#5 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(109): App_Controller->__loadView()
#6 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(90): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array)
#7 C:\xampp\htdocs\default-app\lib\App\Router.php(117): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#8 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#9 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#10 {main}
2013-02-22T17:44:46+01:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:136
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(115): App_Router_Router->findRoute('/default-app\mo...')
#1 C:\xampp\htdocs\default-app\lib\App\View.php(69): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\app\views\index\index.phtml(1): App_View->action('test', 'moep')
#3 C:\xampp\htdocs\default-app\lib\App\View.php(40): require_once('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\default-app\lib\App\Controller.php(94): App_View->loadView('index', 'index')
#5 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(109): App_Controller->__loadView()
#6 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(90): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array)
#7 C:\xampp\htdocs\default-app\lib\App\Router.php(117): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#8 C:\xampp\htdocs\default-app\lib\App.php(254): App_Router->route(Object(App_Request))
#9 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#10 {main}
2013-03-05T21:29:28+00:00 ERR (3): exception 'Router_ClassFileNotFoundException' with message 'Class file not found' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php:83
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(113): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(250): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(25): App::run()
#3 {main}
2013-03-06T00:05:27+00:00 ERR (3): exception 'Router_ClassFileNotFoundException' with message 'Class file not found' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php:83
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(114): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(250): App_Router->route(Object(App_Request))
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(25): App::run()
#3 {main}
2013-03-06T00:05:31+00:00 ERR (3): exception 'App_Exception' with message 'Layout template does not exists: default.phtml' in /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/View.php:69
Stack trace:
#0 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Controller.php(97): App_View->loadLayoutView('index', 'index')
#1 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php(116): App_Controller->__loadView()
#2 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router/Dispatcher.php(97): App_Router_Dispatcher->dispatchController('Index', 'indexAct', Array)
#3 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App/Router.php(114): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#4 /Applications/XAMPP/xamppfiles/htdocs/default-app/lib/App.php(250): App_Router->route(Object(App_Request))
#5 /Applications/XAMPP/xamppfiles/htdocs/default-app/public/index.php(25): App::run()
#6 {main}
2013-09-03T16:40:45+02:00 ERR (3): exception 'Router_BadClassNameException' with message 'Disallowed characters in class name PhpStorm-c88fcb18-6455-4fe7-ab58-f8516b7e3f9a.php' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:74
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(248): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-03T16:40:45+02:00 ERR (3): exception 'Router_BadClassNameException' with message 'Disallowed characters in class name PhpStorm-c88fcb18-6455-4fe7-ab58-f8516b7e3f9a.php' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:74
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(248): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-03T17:26:31+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(124): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(263): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-05T18:32:35+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(124): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(263): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-05T18:51:05+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(124): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(265): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-05T19:40:53+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(125): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(263): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-05T20:31:07+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(125): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(270): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:34:12+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(125): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(270): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:41:18+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(125): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(270): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:45:23+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(125): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:48:04+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:51:45+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:53:57+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:53:59+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:54:00+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:58:06+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:58:10+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:58:13+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:58:23+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:58:25+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:59:17+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/in...')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T19:59:20+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-appmoe...')
#1 C:\xampp\htdocs\default-app\lib\App\View.php(96): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\app\views\index\index.phtml(3): App_View->action('test', 'moep')
#3 C:\xampp\htdocs\default-app\lib\App\View.php(44): require_once('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\default-app\lib\App\View.php(27): App_View->loadView('index', 'index')
#5 C:\xampp\htdocs\default-app\app\layout\default.phtml(7): App_View->_getContent()
#6 C:\xampp\htdocs\default-app\lib\App\View.php(70): require_once('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\default-app\lib\App\Controller.php(102): App_View->loadLayoutView('index', 'index')
#8 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(119): App_Controller->__loadView()
#9 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(100): App_Router_Dispatcher->dispatchController('Index', 'indexAction', Array)
#10 C:\xampp\htdocs\default-app\lib\App\Router.php(128): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#11 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#12 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#13 {main}
2013-09-26T19:59:21+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-appmoe...')
#1 C:\xampp\htdocs\default-app\lib\App\View.php(96): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\app\views\index\index.phtml(3): App_View->action('test', 'moep')
#3 C:\xampp\htdocs\default-app\lib\App\View.php(44): require_once('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\default-app\lib\App\View.php(27): App_View->loadView('index', 'index')
#5 C:\xampp\htdocs\default-app\app\layout\default.phtml(7): App_View->_getContent()
#6 C:\xampp\htdocs\default-app\lib\App\View.php(70): require_once('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\default-app\lib\App\Controller.php(102): App_View->loadLayoutView('index', 'index')
#8 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(119): App_Controller->__loadView()
#9 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(100): App_Router_Dispatcher->dispatchController('Index', 'indexAction', Array)
#10 C:\xampp\htdocs\default-app\lib\App\Router.php(128): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#11 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#12 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#13 {main}
2013-09-26T19:59:25+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-appmoe...')
#1 C:\xampp\htdocs\default-app\lib\App\View.php(96): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\app\views\index\index.phtml(3): App_View->action('test', 'moep')
#3 C:\xampp\htdocs\default-app\lib\App\View.php(44): require_once('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\default-app\lib\App\View.php(27): App_View->loadView('index', 'index')
#5 C:\xampp\htdocs\default-app\app\layout\default.phtml(7): App_View->_getContent()
#6 C:\xampp\htdocs\default-app\lib\App\View.php(70): require_once('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\default-app\lib\App\Controller.php(102): App_View->loadLayoutView('index', 'index')
#8 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(119): App_Controller->__loadView()
#9 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(100): App_Router_Dispatcher->dispatchController('Index', 'indexAction', Array)
#10 C:\xampp\htdocs\default-app\lib\App\Router.php(128): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#11 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#12 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#13 {main}
2013-09-26T19:59:33+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-appmoe...')
#1 C:\xampp\htdocs\default-app\lib\App\View.php(96): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\app\views\index\index.phtml(3): App_View->action('test', 'moep')
#3 C:\xampp\htdocs\default-app\lib\App\View.php(44): require_once('C:\xampp\htdocs...')
#4 C:\xampp\htdocs\default-app\lib\App\View.php(27): App_View->loadView('index', 'index')
#5 C:\xampp\htdocs\default-app\app\layout\default.phtml(7): App_View->_getContent()
#6 C:\xampp\htdocs\default-app\lib\App\View.php(70): require_once('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\default-app\lib\App\Controller.php(102): App_View->loadLayoutView('index', 'index')
#8 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(119): App_Controller->__loadView()
#9 C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php(100): App_Router_Dispatcher->dispatchController('Index', 'indexAction', Array)
#10 C:\xampp\htdocs\default-app\lib\App\Router.php(128): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#11 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#12 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#13 {main}
2013-09-26T20:00:31+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/in...')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T20:00:47+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T20:00:49+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T20:01:30+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T20:01:45+02:00 ERR (3): exception 'Router_ClassMethodNotFoundException' with message 'method not found testAction' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:96
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(128): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T20:01:50+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T20:01:53+02:00 ERR (3): exception 'Router_ClassFileNotFoundException' with message 'Class file not found' in C:\xampp\htdocs\default-app\lib\App\Router\Dispatcher.php:86
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(128): App_Router_Dispatcher->dispatch(Object(App_Router_Route), Object(App_Request))
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2013-09-26T20:10:57+02:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\default-app\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\default-app\lib\App\Router.php(126): App_Router_Router->findRoute('/default-app/')
#1 C:\xampp\htdocs\default-app\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\default-app\public\index.php(24): App::run()
#3 {main}
2014-01-17T15:40:16+01:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\Skelappton\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\Skelappton\lib\App\Router.php(125): App_Router_Router->findRoute('/Skelappton/')
#1 C:\xampp\htdocs\Skelappton\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\Skelappton\public\index.php(24): App::run()
#3 {main}
2014-01-17T15:40:20+01:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\Skelappton\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\Skelappton\lib\App\Router.php(125): App_Router_Router->findRoute('/skelappton/')
#1 C:\xampp\htdocs\Skelappton\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\Skelappton\public\index.php(24): App::run()
#3 {main}
2014-01-17T15:40:23+01:00 ERR (3): exception 'Router_RouteNotFoundException' in C:\xampp\htdocs\Skelappton\lib\App\Router\Router.php:143
Stack trace:
#0 C:\xampp\htdocs\Skelappton\lib\App\Router.php(125): App_Router_Router->findRoute('/skelappton/')
#1 C:\xampp\htdocs\Skelappton\lib\App.php(253): App_Router->route(Object(App_Request))
#2 C:\xampp\htdocs\Skelappton\public\index.php(24): App::run()
#3 {main}
